src/plugins/visualizations/public/wizard/group_selection/group_selection.tsx

```diff
@@ -227,7 +227,11 @@ const ToolsGroup = ({ visType, onVisTypeSelected, showExperimental }: VisCardPro
       <EuiFlexItem>
         <EuiFlexGroup gutterSize="s" alignItems="center" responsive={false}>
           <EuiFlexItem grow={false}>
-            <EuiLink data-test-subj={`visType-${visType.name}`} onClick={onClick}>
+            <EuiLink
+              data-test-subj={`visType-${visType.name}`}
+              data-vis-stage={visType.stage}
+              onClick={onClick}
+            >
```

Comment:
Need to be able to tell the stage for **all** visualizations in the "Create visualization" menu for the tests that I added below, so I added the `data-vis-stage` attribute to the link following the pattern of the `EuiPanel` above.